package.json

```diff
@@ -9,8 +9,8 @@
   ],
   "scripts": {
     "start": "npm run start:testrpc",
-    "start:testrpc": "ganache-cli -p 7545 -i 5777",
-    "migrate": "truffle migrate",
+    "start:testrpc": "ganache-cli -p 7545 -i 5777 -a 61 -m 'drastic outer student holiday hour ordinary reduce absurd clever govern ensure merry keen sound pelican'",
+    "migrate": "truffle migrate --reset && node data/index.js",
```

Comment:
Maybe make a separate task that seeds, aka "migrate:dev"?

Reply:
@MaxBlaushild  Cool changed and cleaned up functions